tools/ScenarioTest.ResourceManager/Constants.cs

```diff
@@ -85,6 +85,9 @@ public class Category
         //Azure NRP Firewall dev team
         public const string azurefirewall = "azurefirewall";
 
+        //Azure NRP bastion dev team
+        public const string bastion = "bastion";
```

Comment:
Just to clarify, this should be the same as your team's alias

Reply:
Yes team alias is bastion@microsoft.com